pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAlternativeTitlesTest.scala

```diff
@@ -79,6 +79,20 @@ class SierraAlternativeTitlesTest
     getAlternativeTitles(varFields) shouldBe List("A", "B")
   }
 
+  it("should not include a subfield tagged 5 with content UkLW") {
+    val varFields = List(
+      createVarFieldWith(
+        "246",
+        "1",
+        List(
+          MarcSubfield(tag = "a", content = "A"),
```

Comment:
Given that `A` could be a title or a tag, maybe we could pick a test string that makes it clearer this is a title?

e.g.

```suggestion
          MarcSubfield(tag = "a", content = "Another title for my book"),
```

Reply:
This was following the convention already in the file - I'll change the whole thing